### PR TITLE
New RFC: command-line flag for slurping

### PR DESCRIPTION
**Pull request by xenu**
It's not clear to me what's the process to allocate an RFC number, so I set it to XX for now.

Pre-RFC discussion: https://www.nntp.perl.org/group/perl.perl5.porters/2021/09/msg261616.html

### CODE REVIEW

**Comment by neilb** [comment]
You get **0011**.

**Comment by xenu** [comment]
Updated.

File: rfcs/rfc0011.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    Author: Tomasz Konojacki <me@xenu.pl>
+    Sponsor:
+    ID: 0011
+    Status: Draft
+
+## Abstract
+
+Introduce a new command-line flag for `perl`, `-g`, which sets `$/` to `undef`,
+and thus enables slurp mode.
```
Comment (oodler577) [inline rfcs/rfc0011.md:13]: It would be helpful to note this is to create a flag that is equivalent to `-0777` (only mented below).

**Comment by oodler577** [comment]
Any support to changing this from `-g` (common for debugging via `gcc`) with `-G`? The capitalization would indicate something special about it (it is a _special case_ of `-0`) and as far as I know, it's not used as a _defacto_ flag for anything. Thank you for writing up the RFC and for the consideration of this suggestion.

**Comment by xenu** [comment]
Collisions with other programs' single-letter flags are unavoidable. I don't think it's something worth worrying about.

File: rfcs/rfc0011.md (hunk 1 above)
Comment (xenu) [reply rfcs/rfc0011.md]: Done.

**Review by leonerd — APPROVED** [review]
LGTM

**Pull request merged**